### PR TITLE
Default FelixConfiguration RouteTableRange when using AWS CNI

### PR DESCRIPTION
**Pull request by petercork**
Cherry-pick to 1.20 for open source. 

* Move GKE ROUTETABLERANGE to be set like for AWS
* Move FelixConfig to watch for all variants

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Comment by tmjd** [comment]
This is a pick of #1447

File: pkg/controller/installation/core_controller.go [hunk 1]
```diff
@@ -259,6 +265,12 @@ func add(mgr manager.Manager, r *ReconcileInstallation) error {
 		if err != nil {
 			return fmt.Errorf("tigera-installation-controller failed to watch FelixConfiguration resource: %w", err)
 		}
+
+		//watch for change to primary resource LogCollector
+		err = c.Watch(&source.Kind{Type: &operator.LogCollector{}}, &handler.EnqueueRequestForObject{})
```
Comment (tmjd) [inline pkg/controller/installation/core_controller.go:270]: This was not in the original pick #1447, was this needed?

**Review by tmjd — CHANGES_REQUESTED** [review]
There is one addition that I think we should leave out since it wasn't part of the original PR.

File: pkg/controller/installation/core_controller.go (hunk 1 above)
Comment (petercork) [reply pkg/controller/installation/core_controller.go:270]: argh, there were merge conflicts and this was not meant to be there. Good spot, i will remove

**Pull request closed without merging**